src/utils/files.ts

```diff
@@ -0,0 +1,2 @@
+export const getFileExt = (fileName: string): string | undefined =>
```

Comment:
correct me if I am wrong, but as long as fileName is of type `string`, fileName.split(".").shift() will **_not_** gibe us an undefined value right?

Reply:
good catch, changed the return type!